src/stream/src/executor/hash_agg.rs

```diff
@@ -592,93 +594,87 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
     }
 }
 
+/// NOTE(kwannoel): This should only be used by `test` or `bench`.
+#[allow(clippy::too_many_arguments)]
+pub async fn new_boxed_hash_agg_executor<S: StateStore>(
```

Comment:
What about move this function to `src/stream/src/executor/test_utils.rs`, to make it together with `new_boxed_simple_agg_executor`.